app/Main.hs

```diff
@@ -10,16 +10,81 @@
 module Main ( main ) where 
 
 import System.IO
+import Data.Char (toUpper)
 
--------------------------------------------------------------------------------
+-- Temporary Definitions - When Card/Deck are defined in the library this will be
+-- removed
+type Card = String
+type Deck = [Card]
+
+loadDeck :: IO (Maybe Deck)
```

Comment:
It may be nice to add [haddock comments](https://www.haskell.org/haddock/doc/html/ch03s08.html) for all these, e.g. 
```suggestion
-- | 'loadDeck' is a computation which tries to load a deck from disk.
loadDeck :: IO (Maybe Deck)
```

Reply:
I was planning to do that when I implemented them, they were just placeholders to make the code compile for now. I'll write those in in a second